src/toil/jobStores/fileJobStore.py

```diff
@@ -163,10 +164,12 @@ def deleteFile(self, jobStoreFileID):
         
     def fileExists(self, jobStoreFileID):
```

Comment:
I would make this method protected (by calling it _fileExists) and add a doc string describing that it atomically does two things: 1) tests whether a file exists and 2) asserts that it is a file.